packages/extension-commerce-bm-seo/README.md

```diff
@@ -0,0 +1,124 @@
+
+- Integrate routes managed in Business Manager with the PWA Kit
+- Integration with SCAPI Shopper SEO URL Mapping API
+- Support for multiple resource types (products, categories)
+
+## Installation
+
+```sh
+ npm install @salesforce/extension-commerce-bm-seo
+```
+Also: 
+ - install the peer dependencies listed in the package.json
+ - see the Peer Dependancies section below for any other steps (e.g. make sure your app uses CommerceApiProvider component)
+
+## Peer Dependencies
+
+PWA-Kit Application Extensions are NPM packages at their most simplest form, and as such you can define
+what peer dependencies are required when using it. Because this application extension provides
+Chakra UI via a page and components, it requires that the some peer dependencies are installed.
+
+Depending on what features your application extensions provides it's recommended you include any third-party
+packages as peer dependencies so that your base application doesn't end up having multiple versions of a 
+given package. See package.json for the full list of peer dependencies.
+
+## Configuration
+
+The SEO extension is configured via the `mobify.app.extensions` property in your config files or `package.json`:
+```json
+{
+    "mobify": {
+        "app": {
+            "extensions": [
+                [
+                "@salesforce/extension-commerce-bm-seo",
+                {
+                    "enabled": true,
+                    "commerceAPI": {
+                    "proxyPath": "/mobify/proxy/api",
+                    "parameters": {
+                        "shortCode": "8o7m175y",
+                        "clientId": "c9c45bfd-0ed3-4aa2-9971-40f88962b836",
+                        "organizationId": "f_ecom_zzrf_001",
+                        "siteId": "RefArchGlobal"
+                    }
+                    },
+                    "commerceAPIAuth": {
+                    "propertyNameInLocals": "commerceAPIAuth"
+                    },
+                    "resourceTypeToComponentMap": {
+                    "category": "ProductList",
+                    "product": "ProductDetail",
+                    }
+                }
+                ]
+            ]
+        }
+    }
+}
+```
+
+Please note that the this extension needs to be defined in the array before any other extension that uses `isNavigationBlocked` to
+toggle the rendering of `<WrappedComponent />` in the extension's HOC. Otherwise, `extension-commerce-bm-seo` will not be able to
+set `isNavigationBlocked` back to a state to allow the rendering of `<WrappedComponent />` since its logic is within the component. 
+
+### Configuration Options
+
+- `commerceAPI`: Settings for connecting to the Commerce API
```

Comment:
We probably want to mention that this configuration is optional. We might want to mention that if a commerce provider is already added from another extension, that instance will be used.